manufacturers/rules.stanza

```diff
@@ -37,11 +45,15 @@ public pcb-rules bay-area-circuits-conservative-rules :
   min-copper-edge-space = 0.381 ; 10mil outer, but 15mil inner
   min-annular-ring = 0.1524 ; 6mil
   min-drill-diameter = 0.254 ; 10mil
-  min-silkscreen-width = 0.0762 ; 3mil
-  min-pitch-leaded = 0.35 ; guess
-  min-pitch-bga = 0.35 ; guess
+  min-silkscreen-width = 0.0762 ; 3mil ;
```

Comment:
You can't comment the new line character xD

Reply:
good catch!